rbd/options.go

```diff
@@ -33,16 +33,30 @@ type RbdImageOptions struct {
 
 type RbdImageOption C.int
 
+// NewRbdImageOptions creates a new RbdImageOptions struct. Call
+// RbdImageOptions.Destroy() to free the resources.
+//
+// Implements:
+//  void rbd_image_options_create(rbd_image_options_t* opts)
 func NewRbdImageOptions() *RbdImageOptions {
 	rio := &RbdImageOptions{}
 	C.rbd_image_options_create(&rio.options)
 	return rio
 }
 
+// Destroy a RbdImageOptions struct and free the associated resources.
+//
+// Implements:
+//  void rbd_image_options_destroy(rbd_image_options_t opts);
 func (rio *RbdImageOptions) Destroy() {
 	C.rbd_image_options_destroy(rio.options)
 }
 
+// SetString the value of the RbdImageOption to the given string.
```

Comment:
Usually for these function names that are not "verb like", such as destroy above, I resort to treating it like a proper name instead of trying make it read like two words. I suggest something like "SetString changes the value of the RbdImageOption to the given string."

Reply:
Ok, I have updated this.